tests/missing_cces.py

```diff
@@ -41,12 +58,16 @@ def check_all_rules(root):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(
         description="Checks the all rules that are part of at least one "
-        "profile have a CCE assigned.")
+        "profile have a CCE assigned."
+        "Optionally with -p provide comma-separated profile list to check against.")
     parser.add_argument(
         "datastream_path", help="Path to a SCAP source datastream")
+    parser.add_argument(
+        "-p", help="Comma-separated list to check for missing CCEs",
```

Comment:
It would be great to add also the long option eg. `--profiles` That would be consistent because we usually provide them. Also using them helps understandably of the commands.

Reply:
added in [96c4dff](https://github.com/ComplianceAsCode/content/pull/9155/commits/96c4dff87ea430928d5e8e9c4f7ecccf7fbe7356)